public/app/features/alerting/unified/MuteTimings.test.tsx

```diff
@@ -143,169 +155,135 @@ const defaultConfigWithBothTimeIntervalsField: AlertManagerCortexConfig = {
   template_files: {},
 };
 
-const resetMocks = () => {
-  jest.resetAllMocks();
+const expectToHaveRedirectedToTable = async () => expect(await screen.findByText(indexPageText)).toBeInTheDocument();
```

Comment:
nit: I would change this variable name as its not a table Something like `expectToHaveRedirectedToRightRoute`